cypress.json

```diff
@@ -1,4 +1,11 @@
 {
+  "env": {
+    "runAudit": true
+  },
   "defaultCommandTimeout": 10000,
-  "baseUrl": "http://localhost:5001/"
+  "baseUrl": "http://localhost:5001/",
+  "video": false,
+  "lighthouse": {
+    "accessibility": 90
```

Comment:
I think we were going with `">75"`
maybe will be better to check this with Sarah

Reply:
I suggested we use that threshold of 75. But that was just as an initial aim and open for revision. I'm happy for us to go with 90 as we can reassess later.

Reply:
Hey, I had a good look around other projects that use some sort of accessibility reporting and the benchmark they use is generally 95+ so I thought we should be aiming for at least 90. It will be good practice to add any accessibility tweaks from an early stage and it seems so far that we are way above this 75 mark anyway